src/sentry/static/sentry/app/components/contextPickerModal.tsx

```diff
@@ -215,6 +216,24 @@ class ContextPickerModal extends React.Component<Props> {
     });
   };
 
+  //TODO(TS): Fix typings
+  customOptionProject = ({label, ...props}: any) => {
+    const project = this.props.projects.find(({slug}) => props.value === slug);
+    if (!project) {
+      return null;
+    }
+    return (
+      <components.Option label={label} {...props}>
```

Comment:
Is `Option` not exported directly?

Reply:
@billyvg It is not. `react-select` just exports `components`.